CONTRIBUTING.md

```diff
@@ -5,12 +5,13 @@ Contribution to this project is very much like contributing to any other Free/Li
 3. Make a new branch from `develop` branch (This is important here! You need to checkout to `develop` branch and then make a branch)
 4. Make the local development environment ready by running `npm install`.
 5. To run the project on live server, run `npm run watch`.
-6. Commit your changes.
-7. Push your commits to your own fork
-8. Send Pull Request from your new branch to this project's **develop branch**.
+6. To run the project on local host, run `npm run build` then `npm start`.
+7. Commit your changes.
+8. Push your commits to your own fork
+9. Send Pull Request from your new branch to this project's **develop branch**.
```

Comment:
Please undo these changes and send them as a separate PR.

Please try to keep PR and its commits all relevant.